packages/calypso-e2e/src/media-helper.ts

```diff
@@ -66,7 +66,8 @@ export function createTestFile( {
 	sourceFileName: string;
 	testFileName?: string;
 } ): string {
-	let fileName = getTimestamp();
+	let fileName = `${ getTimestamp() }${ getRandomInteger( 100, 999 ) }`;
```

Comment:
I was thinking about this method of generating names, and wondered: do we want to separate the integer from the timestamp piece? In case we need to make sense of that timestamp piece later?

In other words, it would be something like: `${ getTimestamp() }x${ getRandomInteger( 100, 999 ) }`